views/inc_qualityChooser.mako

```diff
@@ -43,14 +43,24 @@ selected = None
         <h5><b>Quality setting explanation:</b></h5>
         <h5 id="allowed_text">This will download <b>any</b> of these qualities and then stops searching: <label id="allowed_explanation">${', '.join([Quality.qualityStrings[i] for i in allowed_qualities])}</label></h5>
         <h5 id="preferred_text1">Downloads <b>any</b> of these qualities: <label id="allowed_preferred_explanation">${', '.join([Quality.qualityStrings[i] for i in allowed_qualities + preferred_qualities])}</label></h5>
-        <h5 id="preferred_text2">But it will stop searching when one of these is downloaded:  <label id="preferred_explanation">${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</label></h5>   
+        <h5 id="preferred_text2">But it will stop searching when one of these is downloaded:  <label id="preferred_explanation">${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</label></h5>
     </div>
     <div>
         <h5 class="red-text" id="backlogged_episodes"></h5>
     </div>
     <div id="archive" style="display: none;">
-        <h5><b>You can archive episodes with 'downloaded' final status to avoid creating backlog:</b></h5>
+        <h5>
+            <b>
+                Archive all currently downloaded episodes to avoid increasing
```

Comment:
I tried some different wordings and none really worked for me.... I think for now I'll just remove the word all, and leave further clarification for later.